README.md

```diff
@@ -78,6 +78,9 @@ Clone infrastructure repository:
 
 NOTE: VM will boot from volume, you no longer have to attach additional volumes. Size of the volume can be set by `-e vm_size=100`
 
+    Install the various ansible roles
+    (dev) $ ansible-galaxy install -r requirements.yml
```

Comment:
there is no need to install them all, how about if we say `ansible-galaxy install openmicroscopy.devspace`